qiskit/circuit/quantumcircuit.py

```diff
@@ -1256,77 +1246,13 @@ def draw(self, output=None, scale=None, filename=None, style=None,
             fontsize (int): The font size to use for text. Defaults to 13.
             subfontsize (int): The font size to use for subtext. Defaults to 8.
             displaytext (dict): A dictionary of the text to use for each
-                element type in the output visualization. The default values
-                are::
-
-                    {
-                        'id': 'id',
-                        'u0': 'U_0',
-                        'u1': 'U_1',
-                        'u2': 'U_2',
-                        'u3': 'U_3',
-                        'x': 'X',
-                        'y': 'Y',
-                        'z': 'Z',
-                        'h': 'H',
-                        's': 'S',
-                        'sdg': 'S^\\dagger',
-                        't': 'T',
-                        'tdg': 'T^\\dagger',
-                        'rx': 'R_x',
-                        'ry': 'R_y',
-                        'rz': 'R_z',
-                        'reset': '\\left|0\\right\\rangle'
-                    }
-
+                element type in the output visualization.
+                See :class:`~qiskit.visualization.qcstyle.DefaultStyle` for the defaults.
```

Comment:
I don't actually think this is a valid docs link right now. I'm not able to find this path in the documentation https://qiskit.org/documentation/apidoc/visualization.html it probably means we need to add this to the docs first.

Reply:
That just means adding it in the `__init__` docstring or do I also need to create a new apidocs file?

Reply:
I referenced the module for now, and added the module to the `__init__`

Reply:
Yeah, you just need it in the `qiskit.visualization` `__init__` module. There's no need to add a new apidocs.rst unless you want to create a new top page in the docs tree.  (which for qcstyle I don't think makes sense, at least not yet until we improve the interface around it).

Reply:
I think we might need to add some additional setup or docstrings to make this work. Looking at the rendered output from sphinx of this it doesn't contain any real details:

![Screenshot_2020-10-21_10-42-21](https://user-images.githubusercontent.com/2447371/96736068-39bc7a00-138a-11eb-82fd-64857bf54adc.png)